fuzzers/glibfuzzer/builder.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image
+FROM $parent_image
+
+#RUN git clone https://github.com/llvm/llvm-project.git /llvm-project && \
+#RUN  git clone https://github.com/gtt1995/libfuzzer-adaptive-group.git&& \
+RUN  git clone https://github.com/gtt1995/libfuzzer-cmab-latest.git && \
```

Comment:
This repo no longer exists.

Reply:
Hello, now this git repo can be accessed.
https://github.com/gtt1995/libfuzzer-cmab-latest